test/unit/dimensions.js

```diff
@@ -461,7 +462,9 @@ test( "allow modification of coordinates argument (gh-1848)", 1, function() {
 		return coords;
 	});
 
-	equal( element.offset().top, 100, "coordinates are modified" );
+	offsetTop = element.offset().top;
+	ok( Math.abs(offsetTop - 100) < 0.02,
```

Comment:
I have no idea of a better number, so we might as well go with yours. It seems like this may be an issue we will see more often if we let more values be non-integer ones.